browser/net/BUILD.gn

```diff
@@ -8,6 +8,8 @@ source_set("net") {
   sources = [
     "brave_ad_block_tp_network_delegate_helper.cc",
     "brave_ad_block_tp_network_delegate_helper.h",
+    "brave_block_reporting_urls.cc",
+    "brave_block_reporting_urls.h",
```

Comment:
the list should be sorted... And maybe it's better to name it `brave_block_safebrowsing_urls.h`?